modules/defining-running-jobs.adoc

```diff
@@ -0,0 +1,90 @@
+// Module included in the following assemblies:
+//
+// * /develop/running-kueue-jobs.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="defining-running-jobs_{context}"]
+= Defining a job to run with {product-title}
+
+When you are defining a job to run with {product-title}, ensure that it meets the following criteria:
+
+* Specify the local queue to submit the job to, by using the `kueue.x-k8s.io/queue-name` label.
+* Include the resource requests for each job pod.
```

Comment:
Are these prerequisites? It seems that the local queue criteria is repeated as a prereq

Reply:
No - a local queue being created is a prereq. This line is about pointing to that local queue in the job YAML.